CMakeLists.txt

```diff
@@ -78,19 +78,19 @@ enable_language(C)  # C needed even for basic platform introspection
 
 # Set install paths ============================================================
 
-set(TILEDARRAY_INSTALL_BINDIR "bin"
+set(TILEDARRAY_INSTALL_BINDIR "${CMAKE_INSTALL_BINDIR}"
```

Comment:
These should come after `include(GNUInstallDirs)`

Reply:
Thanks, I moved the set of install paths into the place after `include(GNUInstallDirs)`

Reply:
it appears that currently `TILEDARRAY_INSTALL_*` vars are not used by files in `cmake/modules` but were used in the past. So keeping the original location of the definitions of `TILEDARRAY_INSTALL_*` would be ideal.

Thanks, @topazus !